forc/src/cli/help.rs

```diff
@@ -0,0 +1,298 @@
+#[macro_export]
+// Let the user format the help and parse it from that string into arguments to create the unit test
+macro_rules! cli_examples_v2 {
```

Comment:
Referring to something as `v2` and having different functionality is confusing. I would rename this to inform the reader of the difference between the original and this version